bisnode/models.py

```diff
@@ -184,6 +187,133 @@ def create(self, company_report_id, statement):
         self.save()
 
 
+class BisnodeFinancialStatementSwedenReport(BisnodeCompanySubReport):
+
+    company_report = models.ForeignKey(
+        BisnodeCompanyReport,
+        related_name="financial_statements_sweden")
+    account_period = models.DateField()
+    number_of_months_covered = models.PositiveIntegerField(
+        validators=[MinValueValidator(1), MaxValueValidator(12)])
+    total_turnover = MoneyField(null=True, blank=True, default_currency="SEK",
+                                decimal_places=2, max_digits=14)
+    total_operating_expenses = MoneyField(
+        null=True, blank=True, default_currency="SEK",
+        decimal_places=2, max_digits=14)
+    result_after_depreciation = MoneyField(
+        null=True, blank=True, default_currency="SEK",
```

Comment:
@anah you seem to be repeating yourself a lot with the currencies.

Wouldn't it be better to make a constant?